src/constants.js

```diff
@@ -463,6 +463,16 @@ export const TASK_MAX_GROUP_ASSIGNEES: 250 = 250;
 export const COMMENT_STATUS_OPEN: 'open' = 'open';
 export const COMMENT_STATUS_RESOLVED: 'resolved' = 'resolved';
 
+/* ------------------ Activity Filter Statuses ------------------ */
+export const ACTIVITY_FEED_ITEM_ALL: 'all' = 'all';
+export const ACTIVITY_FEED_ITEM_OPEN: 'open' = 'open';
+export const ACTIVITY_FEED_ITEM_RESOLVED: 'resolved' = 'resolved';
+export const ACTIVITY_FEED_ITEM_TASKS: 'task' = 'task';
```

Comment:
should be plural

Reply:
This currently is not plural so that it matches the activity type for task feed items. This provides the ability to filter with `item.type === feedItemsStatusFilter` instead of first mapping `feedItemsStatusFilter` to a type. I can see how making it plural could be better but that would also require mapping it to `task` for when filtering based on type.

Reply:
I see! In that case, would it help to include a comment about why it's singular here? Would it be better to: 1) use a different variable for the item.type or 2) use the plain string values when filtering via item.type? I'll leave it up to you, but if we keep it as is, maybe a comment would be helpful!